### PR TITLE
Merge edb.eschema into edb.edgeql.

### PR DESCRIPTION
**Pull request by vpetrovykh**

### CODE REVIEW

File: edb/edgeql/declarative.py [hunk 1]
```diff
@@ -29,11 +29,10 @@
 from edb.edgeql import ast as qlast
 from edb.edgeql import codegen as qlcodegen
 from edb.edgeql import compiler as qlcompiler
+from edb.edgeql import parser as edge_parser
```
Comment (1st1) [inline edb/edgeql/declarative.py]: just `parser`

File: edb/edgeql/parser/__init__.py [hunk 2]
```diff
@@ -17,12 +17,12 @@
 #
 
 
-from .parser import EdgeQLExpressionParser, EdgeQLBlockParser
+from . import parser as edge_parser
```
Comment (1st1) [inline edb/edgeql/parser/__init__.py]: just `parser`

File: tests/test_docs.py [hunk 3]
```diff
@@ -23,8 +23,7 @@
 
 from graphql.language import parser as graphql_parser
 
-from edb.edgeql import parser as edgeql_parser
-from edb.eschema import parser as schema_parser
+from edb.edgeql import parser as edge_parser
```
Comment (1st1) [inline tests/test_docs.py]: just `ql_parser`

File: edb/edgeql/declarative.py (hunk 1 above)
Comment (vpetrovykh) [reply edb/edgeql/declarative.py]: ok

File: edb/edgeql/parser/__init__.py (hunk 2 above)
Comment (vpetrovykh) [reply edb/edgeql/parser/__init__.py]: ok

File: tests/test_docs.py (hunk 3 above)
Comment (vpetrovykh) [reply tests/test_docs.py]: ok

**Pull request merged**